pypika/dialects.py

```diff
@@ -36,6 +45,22 @@ def __init__(self, **kwargs: Any) -> None:
         super().__init__(dialect=Dialects.SNOWFLAKE, **kwargs)
 
 
+class SnowflakeCreateQueryBuilder(CreateQueryBuilder):
+    QUOTE_CHAR = None
+    QUERY_CLS = SnowflakeQuery
+
+    def __init__(self) -> None:
```

Comment:
CreateQueryBuilder specifies ALIAS_QUOTE_CHAR=None but SnowflakeQueryBuilder specifies '"'. It is possible this could cause issues in cases where aliases are used in delete/update queries. Would you mind looking at adding the alias quote char to make it consistent with SnowflakeQueryBuilder?